### DISCUSSION TITLE
Derivative-fields and extension-tasks not triggered

### DISCUSSION THREAD

**Discussion by webwurm** [AntlerVC/firetable#1055] [Support / Q&A] [answered]
Suddenly I have weird problems roght now: 

I have a Users-table with some derivative-fields that trigger on change of the "time_created"-field and some tasks that trigger when a new entry is created, like creating welcome-messages and such.

My problem now is: Nothing seems to trigger anymore when creating a new document in the Users-collection; no tasks are working and the derivative-fields are not calculated until I press "Evaluate".

E.g. this one:
![Screenshot 2023-01-04 114836](https://user-images.githubusercontent.com/1053176/210539183-0bf17d8d-0d62-4961-a48e-bd4c5ebca601.jpg)

Is there a general problem right now? It effects many different scripts I have; that's kind of weird.
Thank you!

**Comment by harinij** [comment, marked as answer]
@webwurm we are not seeing an issue like you mentioned and see the derivatives and extensions are working fine. Can you check the cloud logs to see if you see anything odd? Did you do a build of them recently and was it successful?
<img width="407" alt="Screen Shot 2023-01-04 at 11 17 19 pm" src="https://user-images.githubusercontent.com/307298/210553446-14866a8d-0b2b-426e-855e-2f2f09793e1c.png">

**Reply by kudax** [reply]
I am also experiencing the same issue, where all the tasks I created in the last few days are not triggering on create and on update. I have deployed them correctly and repeatedly, but nothing shows on  the logs.

**Reply by bofeiw** [reply]
@kudax @webwurm Do you see anything odd in `rowy-backend` Cloud Run instance? That's where the derivative code will run. The issue might be related to our recent release of Rowy Logging.

https://console.cloud.google.com/run/

**Reply by kudax** [reply]
I have checked the rowy-backend Cloud Run logs and there is nothing untoward.

**Reply by webwurm** [reply]
I tried to deploy everything again, but it didn't change anything; the build was succesfull thow.
The logs don't show anything odd.

**Reply by kudax** [reply]
@webwurm @bofeiw I'm glad to report that my extensions are now working. There might have been an issue with my table configuration.
This is what I did:
1. copied my extensions code to an external editor;
2. deleted the extensions;
3. deleted the table;
4. created a new table and named it with a different table  ID;
5. added the extension to the new table and pasted my code;
6. built; and
7. tested successfully.

I hope this helps.

**Reply by webwurm** [reply]
Good to know. It also seems to work for me again - although I didn't really do anything.

**Comment by webwurm** [comment]
I have the problem again...Actions (on field change) and Tasks (on creating a new document in the "Users"-table) are not triggered anymore. I didn't change anything, but the problem appeared yesterday suddenly. I added logging.log to the functions, but it seems they are not even called.

In the cloud logs I find only this error:
![Error](https://user-images.githubusercontent.com/1053176/213911829-222f5a66-3e1a-4922-9b75-c7eff8b4d8a0.jpg)

This error has sadly a heavy impact on me, since all the scripts that are implemented to set up a new user are not working.
Has anybody an idea how to find the error? Thanks!
